package.json

```diff
@@ -3,8 +3,11 @@
   "packageManager": "yarn@3.3.0",
   "scripts": {
     "setup": "npm install yarn -g --save && yarn set version berry && yarn",
-    "check": "node --check ./src/services/evento.js & ./src/services/pago.js & ./src/services/usuario.js"
-    
+    "check": "node --check ./src/services/evento.js & node --check ./src/services/pago.js & node --check ./src/services/usuario.js &",
+    "test": "node --test"
   },
-  "type": "module"
+  "type": "module",
+  "dependencies": {
+    "@faker-js/faker": "^7.6.0"
```

Comment:
???

Reply:
Es la librería que utilizo para generar los datos, debería de añadirla en el readme o con los issues y commits que le dedico esta bien?

Reply:
En este objetivo prefiero que todo lo que uséis lo hayáis escrito vosotros. Entre otras cosas, tendrás que plantear el problema de generar las cosas y escribir un código que lo haga con determinadas características, lo que te ayudará a entender mejor el problema. Usar una biblioteca externa no te ayuda a nada.

> Y si lo usas en los tests, tampoco está en el apartado que corresponde.

Reply:
Ya pero los datos que genera son solamente diferentes nombres o números aleatorios, sigo teniendo que generar las cosas, es que lo veo más completo que poner usuarios=Manuel, Juan y Pepe o usar los mismos 10 pagos y que siempre se hagan los tests con estos valores. En cuanto a que no está en el apartado que le corresponde no lo he entendido.

Reply:
Si solo lo usas en test, no es una dependencia de producción.
Insisto que debe ser algo que generes tú.